Tensile/SolutionStructs.py

```diff
@@ -2980,20 +2980,27 @@ def assignDerivedParameters(state):
         reject(state, "int8 doesn't support LocalSplitU")
         return
 
+    # GlobalSplitU doesn't work with some other things:
     if state["GlobalSplitU"] > 1:
-        # GlobalSplitU doesn't work with some other things:
-        if not state["GlobalSplitUSummationAssignmentRoundRobin"] and state["LoopTail"]:
-          reject(state, "GlobalSplitU and LoopTail require SummationAssignmentRoundRobin=True since strongly breaks Tensile kernel architecture")
-          return
+      if not state["GlobalSplitUSummationAssignmentRoundRobin"] and state["LoopTail"]:
+        reject(state, "GlobalSplitU and LoopTail require SummationAssignmentRoundRobin=True since strongly breaks Tensile kernel architecture")
+        return
 
-        # GSU is working for all gemm functions, except for I8II.
-        if (state["ProblemType"]["DataType"].isInt8() or state["ProblemType"]["DataType"].isInt8x4()):  
-          reject(state, "GlobalSplitU is not for I8 or 4xi8 data type.")
-          return
+      supported = \
+        (state["ProblemType"]["DataType"].isSingle()) or \
+        (state["ProblemType"]["DataType"].isDouble() and state["BufferStore"]) or \
+        (state["ProblemType"]["DestDataType"].isInt32()) or \
+        (state["KernelLanguage"] == "Assembly" and
+            (state["ProblemType"]["DataType"].isHalf() and not state["ProblemType"]["HighPrecisionAccumulate"]) or
+            (state["_GlobalAccumulation"])
```

Comment:
I do not quite understand why we need state["_GlobalAccumulation"] here.
Could you explain why it is necessary?

Reply:
I talked about it in the meeting. In short, _GA should be None, for all SingleBuffer cases, except, fp16/bf16. We need _GA here to cover fp16/bf16.

Reply:
And also for cgemm/zgemm multiplebuffer (singlebuffer is not working).